src/app/portlets/dot-edit-content/services/dot-edit-content-html.service.ts

```diff
@@ -296,7 +266,7 @@ export class DotEditContentHtmlService {
         this.addContainerToolbar();
         this.addContentletMarkup();
 
-        this.initDragAndDropContentlets();
+        this.dotDragDropAPIHtmlService.initDragAndDropContext(this.getEditPageDocument());
```

Comment:
Why instead of passing doc around we just don't make a method that return the <script> or whatever we need and we append it to the doc here? I think we should have the doc in just one place (the main service) if possible.

Reply:
... but you need the doc to create the element 

https://github.com/dotCMS/core-web/pull/494/files#diff-1661e25b47cd545049a58805e87cc482L246